lib/SILOptimizer/Mandatory/TFConstExpr.h

```diff
@@ -38,8 +38,6 @@ class SILValue;
 class SymbolicValue;
 enum class UnknownReason;
 
-namespace tf {
```

Comment:
do you also want to rename TFConstExpr.h to sth like ConstExpr.h?

Reply:
I am unsure about what the final location and name of these files will be, so I will defer all moving/renaming until the patch review gives me a good idea of what they will be.